### PR TITLE
Update docs for set_changelog action

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a changelog summary in the Pull Request description @KrauseFx</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: fastlane/lib/fastlane/actions/set_changelog.rb [hunk 1]
```diff
@@ -81,7 +81,8 @@ def self.description
       def self.details
         [
           "This is useful if you have only one changelog for all languages.",
-          "You can store the changelog in `#{default_changelog_path}` and it will automatically get loaded from there. This integration is useful if you support e.g. 10 languages and want to use the same \"What's new\"-text for all languages."
+          "You can store the changelog in `#{default_changelog_path}` and it will automatically get loaded from there. This integration is useful if you support e.g. 10 languages and want to use the same \"What's new\"-text for all languages.",
+          "Defining the version is optional, fastlane will try to automatically detect it"
```
Comment (taquitos) [inline fastlane/lib/fastlane/actions/set_changelog.rb]: `Defining the version is optional, fastlane will try to automatically detect it` + "if one is not provided"

**Review by taquitos — COMMENTED** [review]
Small addition

**Comment by KrauseFx** [comment]
Done

**Review by taquitos — APPROVED** [review]
🎈

**Pull request merged**